clouddq/main.py

```diff
@@ -211,6 +211,14 @@
     is_flag=True,
     default=False,
 )
+@click.option(
+    "--intermediate_table_expiration_hours",
+    help="Configure the bigquery job intermediate tables expiration hours "
+    "in the provided dataset '--gcp_bq_dataset_id' . Expiration time "
+    "defaults to 24 hours if not set. ",
```

Comment:
Remove "Expiration time defaults to 24 hours if not set." and set `show_default` to `true` in `click.option`?

Reply:
incorporated